src/sagas/demo.js

```diff
@@ -0,0 +1,31 @@
+import { put, select, takeEvery } from "redux-saga/effects";
+import { getLogger } from "../utils/logger";
+import { verifyDemoDocumentCompleted, verifyDemoDocumentFailure, getDemoDocument } from "../reducers/demo";
+import { verifyDocument } from "../services/verify";
```

Comment:
we will eventually need to use a different `verifyDocument` function, or modify the interface so that we can "hardcode" the verifier only to be used in ropsten network, ie pass another option argument to tell `verificationBuilder` to build the verifier properly

Reply:
ok updated verifyDocument fn

Reply:
nice